client/src/components/Home.js

```diff
@@ -1,15 +1,14 @@
 import React from "react";
 import "../styles/Home.css";
 import Profiles from "../components/Profiles";
+import { Container, Header } from "semantic-ui-react";
 
 const Home = () => {
   return (
-    <div>
-      <header className="home-header">
-        <h1>MIN - Imece</h1>
-        <Profiles />
-      </header>
-    </div>
+    <Container text style={{ marginTop: "20px" }}>
+      <Header as="h1">MIN - Imece</Header>
+      <Profiles />
+    </Container>
```

Comment:
This could perhaps use some more explanation as to why it has changed in this PR, since it doesn't seem related (Maybe it should be a separate change?). Also the `home-header` style from `Home.css` has been removed so that file should also be removed if this change is include.